packages/runtime/runtime-definitions/package.json

```diff
@@ -56,6 +56,8 @@
   },
   "typeValidation": {
     "version": "2.0.0",
-    "broken": {}
+    "broken": {
+      "TypeAliasDeclaration_AliasResult": {"forwardCompat": false}
+    }
```

Comment:
General question on this: do we have a process to remove these hints at some point? Because once a breaking change makes it to a release, we want to go back to checking for back/forward compat from that point onwardrs, right?

Reply:
they are cleaned up after each version release.